src/boot/categories.toml

```diff
@@ -279,6 +279,33 @@ Crates to help develop software capable of adapting to various \
 languages and regions.\
 """
 
+[language-implementations]
+name = "Language implementations"
```

Comment:
I'm not sure yet if "Language implementations" is the best name for this. Some comments in the original issue proposed "Programming languages", which might be a better fit, but is also quite long. I'm wondering if there is a better alternative to these.

Reply:
I can't think of a better name sadly but I'm not interested in nitpicking this so I'm ok with whatever the popular consensus is :D

Reply:
I think what makes this not fit well is that someone looking for interpreters would look for "interpreter" and if they don't find it they might look for "compiler", but it seems unlikely for them to search for something called "language implementations".

"Transpilers" are ultimately just "Compilers" where input and output language is essentially the same, so we're down to deciding between "Compilers" and "Interpreters". As I mentioned above, someone looking for "Interpreters" might consider looking in the "Compilers" category, but someone looking for a compiler seems less likely to look in a category called "Interpreters".

Due to the logic above I think I would recommend renaming this caregory to "Compilers". It is technically not entirely correct for interpreters, but since we can't seem to be able to find a good category name that includes all of these concepts, I think this is the best we can do now.

Reply:
I've removed the `language-implementations` category and have added two new categories "compilers" and "interpreters"

Reply:
hmm, I don't think we would want to have two dedicated categories. The difference between compilers and interpreters is a somewhat grey area with JIT compilers being built into interpreters and stuff like that.

Reply:
Should I revert to the `language-implementations` category and include `interpreters` and `compilers` as subcategories?

Also JITs are not normally used by themselves are are usually embedded into a larger process i.e. an interpreter runtime. So I don't think that should affect the other categories but instead it may make sense to have a JIT category of its own.

Reply:
As I mentioned above, I think having a single category called "Compilers" which includes compilers, interpreters and transpilers would probably work best.

Reply:
> I would recommend renaming this category to "Compilers".

My bad I didn't see this